test/plugin/test_out_file.rb

```diff
@@ -557,6 +557,42 @@ def parse_system(text)
     check_gzipped_result(path, formatted_lines * 3)
   end
 
+  test 'append when JST' do
```

Comment:
Could you add more test with different time and timezone, e.g. `timekey_zone` is `+09:00` and actual time is `+02:00`